apps/base-docs/docs/tools/oracles.md

```diff
@@ -75,6 +77,17 @@ See [this guide](https://docs.diadata.org/introduction/intro-to-dia-oracles/requ
 
 ---
 
+## Gelato
+
+Gelato VRF (Verifiable Random Function) provides a unique system offering trustable randomness on Base.
+
+See this guide to learn how to get started with [Gelato VRF](https://docs.gelato.network/web3-services/vrf/quick-start).
+
+#### Supported Networks
+
+- Base Mainnet
+- Base Sepolia (Testnet)
+
```

Comment:
@anirudhmakhana 

Can you also please add a divider below the Gelato entry, to match the others?

Reply:
Thanks @wbnns i have made the requested changes!